packages/venia-concept/_buildpack/create.js

```diff
@@ -0,0 +1,152 @@
+const { resolve } = require('path');
+
+const isDebugging = () => !!process.env.DEBUG_PROJECT_CREATION;
+
+function createProjectFromVenia({ fs }) {
+    const toCopyFromPackageJson = [
+        'main',
+        'browser',
+        'dependencies',
+        'devDependencies',
+        'optionalDependencies',
+        'engines'
+    ];
+    const scriptsToCopy = [
+        'build',
+        'build:analyze',
+        'build:prod',
+        'buildpack',
+        'clean',
+        'download-schema',
+        'lint',
+        'prettier',
+        'prettier:check',
+        'prettier:fix',
+        'start',
+        'start:debug',
+        'test',
+        'validate-queries',
+        'watch'
+    ];
+    return {
+        after({ options }) {
+            fs.outputFileSync(
+                resolve(options.directory, 'babel.config.js'),
+                "module.exports = { presets: ['@magento/peregrine'] };\n",
+                'utf8'
+            );
+        },
+        visitor: {
+            'package.json': ({
+                path,
+                targetPath,
+                options: { name, author, npmClient }
+            }) => {
+                const pkgTpt = fs.readJsonSync(path);
+                const pkg = {
+                    name,
+                    private: true,
+                    version: '0.0.1',
+                    description:
+                        'A new project based on @magento/venia-concept',
+                    author,
+                    license: 'UNLICENSED',
+                    scripts: {}
+                };
+                toCopyFromPackageJson.forEach(prop => {
+                    pkg[prop] = pkgTpt[prop];
+                });
+
+                const npmCli = isDebugging() ? 'yarn' : npmClient;
+
+                const toPackageScript =
+                    npmCli === 'yarn'
+                        ? script => script
+                        : script =>
+                              script && script.replace(/yarn run/g, 'npm run');
+                scriptsToCopy.forEach(name => {
+                    pkg.scripts[name] = toPackageScript(pkg.scripts[name]);
```

Comment:
Possible bug: `pkg.scripts` is never populated since `scripts` is not included in `toCopyFromPackageJson`

Reply:
Should be fixed. My bad.